README.md

```diff
@@ -7,39 +7,52 @@ Usage:
 
 ```bash
 git clone https://github.com/gazebosim/jetty_demo
-cd Jetty_Warehouse
+cd jetty_demo/jetty_demo/worlds
 gz sim -v 4 jetty.sdf
 ```
 
-## Development on distrobox
+Or launch it from `[rocker](https://github.com/osrf/rocker?tab=readme-ov-file#installation)`,
 
 ```bash
-distrobox create -n jetty_demo --hostname jetty_demo -i ghcr.io/gazebosim/jetty_demo:main
-distrobox enter jetty_demo
+rocker --x11 -- ghcr.io/gazebosim/jetty_demo:main ros2 launch jetty_demo world.launch.xml
 
-git clone https://github.com/gazebosim/jetty_demo
-cd jetty_demo/Jetty_Warehouse
-gz sim -v 4 jetty.sdf
+# Don't forget to start the sim!
 ```
 
 ### Launching the Open-RMF demo
 
-> [!NOTE]
-> This section is actively being worked on. The instructions now only highlight running existing Open-RMF example demos.
+![](media/jetty_rmf_demo.png)
+
+```bash
+rocker --x11 -- ghcr.io/gazebosim/jetty_demo:main ros2 launch jetty_demo demo.launch.xml
+
+# Don't forget to start the sim!
```

Comment:
This seems important for running the tasks. Maybe move this out side the code block to be more explicit?

e.g. Just have another instruction like:

Don't forget to hit the Play button to run the sim before starting the tasks below.

Reply:
https://github.com/gazebosim/jetty_demo/pull/10/commits/809ff1d2d840ece824728d69f06bd71dd7229917